src/lwc/geBatchGiftEntryHeader/geBatchGiftEntryHeader.js

```diff
@@ -1,10 +1,8 @@
 import { LightningElement, api, wire, track } from 'lwc';
```

Comment:
Remove unused import.

```suggestion
import { LightningElement, api, wire } from 'lwc';
```